lib/vanagon/driver.rb

```diff
@@ -125,6 +173,28 @@ def retry_task(&block)
     end
     private :retry_task
 
+    # *****NOTE:
+    # This function is temporary and should be used with care. It is meant to
+    # facilitate vanagon development and should be deprecated
+    #     Sean M.    3/28/16
+    # Ask (and wait) for user input, note that this
+    # is mostly for devkit purposes
+    #
+    # @param [string] prompt, what to display before ask
+    def continue(&block)
+      answer = $stdin.gets.chomp!
+      if answer =~ /^y$|^yes$/
+        puts "y"
+
+        yield
+      elsif answer =~ /^s$|^skip$/
+        return
```

Comment:
I'd probably move this around slightly to make it a little more useful for end users who might try to use this (even in its "this is gonna die soon" state):

```ruby
def continue(&block)
  # `gets` reads from $stdin by default so we can omit that
  answer = gets.downcase.chomp

  # 100% less regex
  case answer
  when "y", "yes"
    yield
  when "n", "no"
    puts %(Building on "#{@engine.target}" stopped by user)
    exit
  else
    # I didn't see an easy way to get the name of the stage being run, but this
    # has no explicit `return` and provides a clean fall-through in case something
    # goes really, really weird.
    puts "skipping this stage"
  end
end
```